python/tests/test_writer.py

```diff
@@ -501,3 +501,40 @@ def test_writer_with_options(tmp_path: pathlib.Path):
     )
 
     assert table == data
+
+
+def test_try_get_table_and_table_uri(tmp_path: pathlib.Path):
+    data = pa.table({"vals": pa.array(["1", "2", "3"])})
+    table_or_uri = str(tmp_path / "delta_table")
+    write_deltalake(table_or_uri, data)
```

Comment:
Shouldn't this be unnecessary now?
```suggestion
    table_or_uri = tmp_path / "delta_table"
    write_deltalake(table_or_uri, data)
```

Reply:
Nevermind, see below.